pkgs/by-name/gh/gh-branch/package.nix

```
@@ -0,0 +1,33 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  pkgs,
+}:
+
+stdenv.mkDerivation {
+  pname = "gh-branch";
+  version = "0-unstable-2023-12-06";
+
+  src = fetchFromGitHub {
+    owner = "mislav";
+    repo = "gh-branch";
+    rev = "7ed0aff7601dc4162e0cac8835ecd73409d8a009";
+    hash = "sha256-yiRSXU/jLi067i+gBb3cEHTOuo+w3oEVsGL0NN6shl8=";
+  };
+
+  propagatedBuildInput = with pkgs; [
```

There is a typo here
```suggestion
  propagatedBuildInputs = with pkgs; [
```

---

Would rather use wrapper here; unsure if it propagates when configured as a gh extension.

---

@Aleksanaa Could you give some example? Perhaps another gh extension which uses wrapper?